examples/heat_equation/include/heat_ida.h

```diff
@@ -0,0 +1,200 @@
+#include <deal.II/lac/sparsity_tools.h>
```

Comment:
Add 

~~~C++
#ifndef __heat__ida__h
#define __heat__ida__h
....
#endif
~~~
guard for header files.